### ISSUE TITLE
"Amend previous commit?" view should include the previous commit's changes as well as working tree changes

### ISSUE THREAD

**Issue by dbkaplun** [FredrikNoren/ungit#137]
As of 0.1.6 only the working tree changes are visible.

**Comment by benel** [comment]
Indeed! One of the reason to use "git commit --amend" is for example to remove a file (or a few lines from a file) from the last commit. That would be great to be able to do this with your (already) great tool.
